bundle/sagemaker_rl_agent/lib/python3.5/site-packages/markov/multi_agent_coach/multi_agent_level_manager.py

```diff
@@ -304,11 +304,6 @@ def emulate_step_on_trainer(self, transitions: Dict[str, Transition]) -> None:
         # for i in range(self.steps_limit.num_steps):
         # let the agent observe the result and decide if it wants to terminate the episode
         done = [self.agents[agent_name].observe_transition(transition) for agent_name, transition in transitions.items() if transition]
-        
-        for agent_name, transition in transitions.items():
-            if transition:
-                self.agents[agent_name].total_reward_in_current_episode += transition.reward
```

Comment:
This seems to ensure that a value is added to episode reward in Sagemaker, but seems like value in Metrics is double...